dronegen/misc.go

```diff
@@ -14,9 +14,185 @@
 
 package main
 
+import (
+	"fmt"
+	"strings"
+)
+
 func promoteBuildPipeline() pipeline {
 	// TODO: migrate
-	return pipeline{}
+
+	aptPipeline := promoteAptPipeline()
+	return aptPipeline
+}
+
+// This function calls the build-apt-repos tool which handles the APT portion of RFD 0058.
+func promoteAptPipeline() pipeline {
+	testVersion := "6.2.31"
```

Comment:
Can you add a `//TODO: delete this after testing` comment?

Reply:
Updated and removed